### PR TITLE
More date rules

### PR DESCRIPTION
**Pull request by brendt**
We didn't discuss these specific rules, so I added them via a separate PR. Note that these depend on #5 to be merged.

### CODE REVIEW

**Comment by brendt** [comment]
The PR will have merge conflicts when #5 gets merged, I''l fix them than.

File: src/Rules/DateGreaterThan.php [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+
+class DateGreaterThan implements Rule
+{
+    /** @var \Carbon\Carbon */
+    protected $date;
+
+    /** @var bool */
+    protected $orEquals = true;
+
+    public function __construct(Carbon $date)
```
Comment (freekmurze) [inline src/Rules/DateGreaterThan.php]: Also accept the format here, like in the other PR

File: src/Rules/DateGreaterThan.php [hunk 2]
```diff
@@ -0,0 +1,54 @@
...
+    {
+        $this->date = $date->copy();
+    }
+
+    public function includeBoundaries(): DateGreaterThan
+    {
+        $this->orEquals = true;
```
Comment (freekmurze) [inline src/Rules/DateGreaterThan.php]: rename the instance variable to `includeBoundaries`, that's a bit more clear.

File: README.md [hunk 3]
```diff
@@ -124,6 +126,56 @@ public function rules()
...
+}
+```
+
+By default the boundary date is included, you can exclude it by calling `excludeBoundary`.
+
+```php
+(new DateGreaterThan(now()))
+    ->excludeBoundary()
+```
+
```
Comment (freekmurze) [inline README.md]: Also add an example on how to set the date format.

**Comment by freekmurze** [comment]
Closing due to inactivity.

**Pull request closed without merging**